package.json

```diff
@@ -116,27 +116,6 @@
     "tslint-config-airbnb": "5.11.1",
     "tslint-config-prettier": "1.17.0"
   },
-  "jest": {
```

Comment:
Ninja refactor in an unrelated PR? :)

Reply:
We have a lot of errors on the tslint triggered by the extension on vscode. I don't know why, and I didn't really want to dig into it in this PR but indeed, I fixed some stuffs breaking my files through this :)

Reply:
And actually I rolled back the biggest part of it :) About this, it's just moved in the jest.config.js